### PR TITLE
V7 upgrade

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

**Comment by andreasohlund** [comment]
Compiles fine now, will deal with the inspections next

**Comment by andreasohlund** [comment]
@tmasternak should be good for final review now

File: src/ServiceControl.Transports.AzureServiceBus/ForwardingTopologyAzureServiceBusTransport.cs [hunk 1]
```diff
@@ -1,23 +1,8 @@
 namespace ServiceControl.Transports.AzureServiceBus
 {
     using NServiceBus;
-    using NServiceBus.Settings;
-    using NServiceBus.Transport;
-    using NServiceBus.Transport.AzureServiceBus;
 
     public class ForwardingTopologyAzureServiceBusTransport : AzureServiceBusTransport
     {
```
[the remark this answers is not in the document]
Comment (andreasohlund) [reply src/ServiceControl.Transports.AzureServiceBus/ForwardingTopologyAzureServiceBusTransport.cs]: Pretty sure that forwarding is the default, will double check. But I guess the other PR will sort that out anyway?

File: src/ServiceControl.Monitoring/App_Packages/Particular.Licensing/FilePathLicenseStore.cs [hunk 2]
```diff
@@ -1,16 +1,9 @@
 ﻿namespace Particular.Licensing
```
[the remark this answers is not in the document]
Comment (andreasohlund) [reply src/ServiceControl.Monitoring/App_Packages/Particular.Licensing/FilePathLicenseStore.cs]: Pretty sure, @bording do U remember any breaks?

[the remark this answers is not in the document]
Comment (bording) [reply src/ServiceControl.Monitoring/App_Packages/Particular.Licensing/FilePathLicenseStore.cs]: I'm not aware of anything that would be a problem here.

**Comment by bording** [comment]
Taking a quick look at this, we really need to update this to use the new project file format!

**Comment by andreasohlund** [comment]
Agreed, but that could be handled separately? Would you be able to help
out?(knowing that you’re the expert)

**Comment by tmasternak** [comment]
@bording @andreasohlund I would keep the csproj migration as a separate issue.  It's a good candidate for a maintenance release.

**Pull request merged**